tests/foreman/api/test_ansible.py

```diff
@@ -123,3 +129,56 @@ def test_positive_ansible_job_on_host(target_sat, module_org, rhel_contenthost):
     )
     result = target_sat.api.JobInvocation(id=job['id']).read()
     assert result.succeeded == 1
+
+    target_sat.api.Host(id=target_host.id).remove_ansible_role(data={'ansible_role_id': role_id})
+    host_roles = target_host.list_ansible_roles()
+    assert host_roles[0]['name'] != SELECTED_ROLE
+
+
+@pytest.mark.e2e
+@pytest.mark.stream
+@pytest.mark.tier2
+def test_add_and_remove_ansible_role_hostgroup(target_sat):
+    """
+    Test add and remove functionality for ansible roles in hostgroup via cli
```

Comment:
Do you mean API? The `expectedresults` also mentions cli.

Reply:
ah, copypasta! Updated.